Description: Disable compilation of embedded zlib
 This patch disables the compilation of the zlib source included
 by upstream and, instead, uses the system's zlib for the sources.
 .
 The person named in the Author field signed this changelog entry.
Author: Javier Fernandez-Sanguino <jfs@debian.org>
Origin: debian
Forwarded: not-needed
Reviewed-By: Jose Antonio Quevedo Muñoz <joseantonio.quevedo@gmail.com>
Last-Update: 2011-07-30


--- ossec-hids-2.6.orig/src/Config.Make
+++ ossec-hids-2.6/src/Config.Make
@@ -22,6 +22,6 @@ OS_CRYPTO = ${PT}os_crypto/os_crypto.a
 OS_SHARED = ${PT}shared/lib_shared.a
 OS_ROOTCHECK = ${PT}rootcheck/rootcheck_lib.a
 OS_CONFIG = ${PT}config/lib_config.a
-OS_ZLIB = ${PT}os_zlib/os_zlib.c ${PT}external/libz.a
-OS_LINK = ${IEXTRA}
+OS_ZLIB = ${PT}os_zlib/os_zlib.c
+OS_LINK = ${IEXTRA} -I/usr/include -lz
 DBS = cdb/cdb.a
--- ossec-hids-2.6.orig/src/Makeall
+++ ossec-hids-2.6/src/Makeall
@@ -129,10 +129,10 @@ fi
 # Getting values for each action
 if [ "X${ARGV}" = "Xall" ]; then
     DIRECTORIES="${LIBS} ${SOURCES} ${BINARIES}" 
-    DOZLIB="x"
+    DOZLIB=""
 elif [ "X${ARGV}" = "Xlibs" ]; then
     DIRECTORIES="${LIBS} ${SOURCES}"
-    DOZLIB="x"
+    DOZLIB=""
 elif [ "X${ARGV}" = "Xrootcheck" ]; then
     DIRECTORIES="os_xml os_regex os_net ${SOURCES} ${ROOTCHECKBIN}"    
 elif [ "X${ARGV}" = "Xbuild" ]; then
